resources/analysisTools/snvPipeline/vcf_pileup_compare_allin1_basecount.pl

```diff
@@ -176,15 +181,16 @@
 		$ctrC++;
 		# split lines
 		@ctrl = split (/\s+/, $lineC);
+		$current_c_chr = $ctrl[0];
 		$ccoord = $ctrl[1];
-		if ($tcoord == $ccoord)	# matching pair found!
+		if ($tcoord == $ccoord && $current_t_chr eq $current_c_chr)	# matching pair found!
```

Comment:
If I see this correctly, the algorithm takes the two position-sorted VCFs and steps forward the feature -- either tumor or control VCF -- that is expected "earlier" in the algorithm, until it finds a match. Right?

I think, here and elsewhere, it would make more sense and would make the code clearer, if the chromosome comparison came before the coordinate comparison, i.e. 

```suggestion
		if ($current_t_chr eq $current_c_chr && $tcoord == $ccoord)	# matching pair found!
```